src/ApprenticeFeedback/SFA.DAS.ApprenticeFeedback.Api.UnitTests/Controllers/WhenPuttingTriggerUpdate.cs

```diff
@@ -0,0 +1,32 @@
+﻿using FluentAssertions;
+using MediatR;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+using SFA.DAS.ApprenticeFeedback.Api.Controllers;
+using SFA.DAS.ApprenticeFeedback.Application.Commands.TriggerFeedbackTargetUpdate;
+using SFA.DAS.Testing.AutoFixture;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.ApprenticeFeedback.Api.UnitTests.Controllers
+{
+    public class WhenPuttingTriggerUpdate
+    {
+        private Mock<IMediator> _mockMediator;
+        private ApprenticeFeedbackTargetController _controller;
+
+        [SetUp]
+        public void Arrange()
+        {
+            _mockMediator = new Mock<IMediator>();
+            _controller = new ApprenticeFeedbackTargetController(_mockMediator.Object, Mock.Of<ILogger<ApprenticeFeedbackTargetController>>());
+        }
+
+        [Test, MoqAutoData]
+        public async Task And_CommandIsProcessedSuccessfully_Then_ReturnOk(TriggerFeedbackTargetUpdateCommand request)
```

Comment:
Success test ok, missing one for internal server error if thrown.

Also need to consider if we care that the result has a success boolean and error message and if we want to do anything with it, or leave it to be logged in the controller and let the jobs function output